core/integration/old_core.schema_test.go

```diff
@@ -173,8 +174,17 @@ func TestCoreImageExport(t *testing.T) {
 	// include a random ID so it runs every time (hack until we have no-cache or equivalent support)
 	randomID := identity.NewID()
 	err := engine.Start(context.Background(), nil, func(ctx engine.Context) error {
+		wg := new(sync.WaitGroup)
```

Comment:
What was happening without the WaitGroup? Just curious

Reply:
One of the test runs failed with a race condition due to accessing the `*testing.T` value in `t.Log`. I think the goroutine was sticking around after the test run, so this change makes sure it gets cleaned up after the test is finished.